"Islandora 8" audit

rosiel opened the pull request:
[Edit to add ticket:] https://github.com/Islandora/documentation/issues/1984

## Purpose / why

Islandora 8 appears a lot in the documentation

## What changes were made?

going through some pages to remove "Islandora 8" and replace it with "Islandora" vs Legacy.

## Verification

Does this still make sense?

## Interested Parties


> _**Replace this text** - Name some folks who may be interested, if documentation related mention @Islandora/documentation, or, if unsure, @Islandora/committers_

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [x] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [x] If pages are renamed or removed, have all internal links to those pages been fixed?
* [x] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

rosiel committed:
Edit Editing Docs for Islandora 8 audit.

rosiel committed:
Audit navigation for islandora 8

rosiel committed:
Merge branch 'main' into rosiel-audit-8

seth-shaw-unlv reviewed (changes requested):
Just one little nit-pick, we should be consistent using "Markdown" (this is the form used by [the original creator](https://daringfireball.net/projects/markdown/)) instead of "MarkDown".

Otherwise, this all looks good to me.

rosiel committed:
Style Markdown properly.

rosiel committed:
minor edits to rest-create.

rosiel commented:
> Just one little nit-pick, we should be consistent using "Markdown" (this is the form used by [the original creator](https://daringfireball.net/projects/markdown/)) instead of "MarkDown".
> 
> Otherwise, this all looks good to me.

Don't get me started on OpenSeadragon...

seth-shaw-unlv commented:
So far so good.

rosiel committed:
Rename Islandora 8 a few more places.

rosiel committed:
typo.

rosiel commented:
There are a lot of files to go but I'm going to trim this here and do further PRs for the next ones. (advice would be welcomed, too. It's so tempting to proofread while I have the files open but I might have to stop in the name of getting it done)

ysuarez commented on docs/contributing/editing-docs.md:
```diff
@@ -10,15 +10,17 @@ The [Islandora Documentation Interest Group](https://github.com/islandora-intere
+- optionally, be a member of the Islandora [GitHub Organization](https://github.com/orgs/Islandora/people).
+    - To request an invite, email community@islandora.ca or ask on the Islandora Slack.
+    - This is only needed if you want to commit changes directly to the documentation. Non-members may still make pull requests. 
+- be willing to learn with [Markdown](http://en.wikipedia.org/wiki/Markdown) - a good [Markdown cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) can help.
```
Did you mean to leave the word "with" in line 18? I can make my own PR to fix that and run a CLI spellchecker while I am at it.

ysuarez commented:
Should we add entries in the glossary for "Islandora Legacy" and/or "Islandora 8" too to help others with thsi transition in terms?  I can create a new PR for it or try to push to @rosiel PR?

misilot committed:
Some Islandora 8 to Islandora reference changes

Some Islandora Legacy changes were made in there as well when in the same context as "8"

misilot committed:
Islandora Legacy String Changes

misilot committed:
Whitespace cleanup

misilot committed:
Fix reference to Markdown

misilot committed:
Fix spelling of OpenSeadragon

misilot commented:
@rosiel I opened a [PR](https://github.com/rosiel/documentation/pull/1) against your branch if you want to take a look at that.

misilot committed:
Standardized formatting for TIFF

misilot committed:
Standardize JPEG 2000 spelling, formatting

rosiel commented:
Thanks Tom. The reason I was doing it manually was for catching things like "Islandora 7 and 8" and leaving "Islandora 8" if it's in the title of a recorded youtube video or presentation. I was also doing a mild sanity check as I went, since this update is going to change the "last edited" date. That's why i stopped at 12 files. Let me bring this back to "Draft" status and pull in your changes.

rosiel committed:
Merge pull request #1 from misilot/islandora-8-strings

Islandora 8 strings

misilot commented:
Oh, sorry!

rosiel committed:
Revert to the hard-coded stylings of OpenSeadragon.

rosiel committed:
Mention Legacy Committers.

rosiel committed:
Re-break the styling of Open Seadragon to match UI.

rosiel committed:
Proofread and change some D8 to Drupal.

rosiel committed:
Add entry for Islandora 8 in glossary

rosiel commented:
Thank you so much @misilot ! Your help was a huge time-saver.  @ysuarez I love the suggestion, and just added it.

misilot commented:
Glad it was was helpful in the end! Sorry you had to change "OpenSeadragon" back to "Open Seadragon".  I had gone with the spelling/formatting from https://openseadragon.github.io/

rosiel commented:
That's correct. But as docs, I think they should be descriptive of what the user sees in the UI. That's why I was careful to leave the module name and file formatter as OpenSeadragon while the Context, Media Display Mode, Node Display Mode, and Taxonomy Term, are all as they appear, wrongly, Open Seadragon.

rosiel committed:
Merge branch 'main' into rosiel-audit-8

rosiel committed:
Merge branch 'main' into rosiel-audit-8

rosiel committed:
Merge branch 'main' into rosiel-audit-8

seth-shaw-unlv commented on docs/contributing/committers.md:
```diff
@@ -1,6 +1,6 @@
+# Islandora Committers
 
-Islandora 8 is open source and released under MIT and GPLv2 licenses. The software and associated documentation is developed collectively by a community of contributors and committers. All interested community members are encouraged to contribute to the project. Contributors who demonstrate sustained engagement with the project through quality participation in meetings, mailing lists, documentati ...
+Islandora is open source and released under MIT and GPLv2 licenses. The software and associated documentation is developed collectively by a community of contributors and committers. All interested community members are encouraged to contribute to the project. Contributors who demonstrate sustained engagement with the project through quality participation in meetings, mailing lists, documentation ...
```
Nit-pick: missing a period at the end of the paragraph.

seth-shaw-unlv commented on docs/installation/docker-troubleshooting.md:
```diff
@@ -26,4 +26,4 @@ ERROR: for isle-dc_mariadb_1  UnixHTTPConnectionPool(host='localhost', port=None
 you can try quitting Docker completely (make sure there is no whale icon in your top toolbar - may need to select "Quit" from the whale icon itself) and then restart Docker.
 ### 504 Bad Gateway
-If you get a white screen with a 504 Bad Gateway error, this means your containers haven't finished initializing themselves yet.  If you've waiting an appropriate amount of time (2-5 minutes), then there is most likely an error in a container's startup script.  Use `docker ps -a` to see which services have `Exited` status, and then tail their logs with `docker-compose logs service_name`.
+If you get a white screen with a 504 Bad Gateway error, this means your containers haven't finished initializing themselves yet.  If you've waiting an appropriate amount of time (2-5 minutes), then there is most likely an error in a container's startup script.  Use ` docker ps -a` to see which services have `Exited` status, and then tail their logs with `docker-compose logs service_name`.
```
There shouldn't be a space between "`" and "docker".

seth-shaw-unlv commented:
@rosiel , just two small adjustments then 👍.

misilot commented:
@seth-shaw-unlv I made the changes if you want to review at https://github.com/rosiel/documentation/pull/2

seth-shaw-unlv commented:
That is one way to do it.
